multiselect.go

```diff
@@ -100,7 +100,9 @@ func (m *MultiSelect) OnChange(key rune, config *PromptConfig) {
 				// otherwise just invert the current value
 				m.checked[selectedOpt.Index] = !old
 			}
-			m.filter = ""
+			if config.FilterKeep == false {
```

Comment:
Small nitpick - can we change this to `if !config.FilterKeep`

Reply:
I choose KeepFilter if that's okay with you.

Reply:
That sounds good to me! I'll approve this once you've updated all the things. Thanks again!

Reply:
It was me a pleasure. I do need this feature myself, so it's just developed it.